servers/src/common/adapters.rs

```diff
@@ -531,6 +534,20 @@ impl NetToChainAdapter {
 		}
 	}
 
+	fn clean_rewind_files(&self) {
```

Comment:
Maybe consider moving this to a fn on `txhashset`?

As `adaptor.check_compact()` calls `chain.compact()` which ends up calling `txhashset.compact()`.

That way we don't need to expose `TXHASHSET_SUBDIR` and `OUTPUT_SUBDIR` and we can keep the file related operations confined to `txhashset`.